### ISSUE TITLE
RaidMan integration?

### ISSUE THREAD

**Issue by jonasrembratttetrapak** [smyhk/discord-raidman#1]
Not an issue but a question: 

My virtual DCS squadron ([DCAF](https://dcafoc.wixsite.com/home?fbclid=IwAR1bxTO-EEEQ-e6kRbluSJzJTgauwXJAy-P6n-AZe9V-R6RYxEXy_IXeQ8s)) is using this awesome bot to organize our weekly online events and I am currently writing a bot to apply our policies, such as automatically marking members as "inactive" after not having replied to events for XX weeks, etc.

I was originally planning to use the ability to export the events but is there a better way to instead query the RaidMan bot for these kinds of things?

**Comment by smyhk** [comment]
The email notification I received has your original message before editing and it appears your question is concerning the command prefix. The prefix can be changed to anything you like, but is necessary so bots know when users are interacting with them. You could create a menu system with RichEmbeds that responds to users reactions. The menu could display upon joining a channel, in a DM, user command, etc. You may wish to consult Discord's API documentation as well. Thanks for reaching out and I hope this answers your question.

**Comment by jonasrembratttetrapak** [comment]
I'm still very new to Discord bots and am learning but my question was more if there is a way for one bot to query another, like for the events in this case. I am trying to create a UX that would allow the user to simply run his (new) bot to apply policies based on how his users have reacted (or not reacted) to a series of events. But I guess that's only possible if that other bot (RaidMan in this case) also supports its own API?

**Comment by jonasrembratttetrapak** [comment]
I just checked my original text and the reason I edited it was to avoid misunderstanding. The "!print" command, I am told, is how the RaidMan user gets RaidMan to export all events (to a CSV file). But that would leave him/her with then having to somehow expose that file to this new bot, that would then apply the guild policies. I'm looking for a shortcut, of sort, that avoids the use of files. Sorry if I'm unclear. :-)
